include/deal.II/distributed/tria_base.h

```diff
@@ -464,6 +464,24 @@ namespace parallel
     using CellStatus =
       typename dealii::Triangulation<dim, spacedim>::CellStatus;
 
+    /**
+     * Return true if the triangulation has hanging nodes.
+     *
+     * In the context of parallel distributed triangulations, every
+     * processor stores only that part of the triangulation it locally owns.
+     * However, it also stores the entire coarse mesh, and to guarantee the
+     * 2:1 relationship between cells, this may mean that there are hanging
```

Comment:
I think this needs to be modified because `fullydistributed::Triangulation` does not store the whole coarse mesh. I would phrase this in a way to say that we store a coarse mesh that in general covers a larger part of the computational domain than the locally owned or ghosted cells.